packages/minifiers/minifiers.js

```diff
@@ -88,48 +88,59 @@ CssTools = {
       return path && path.charAt(0) !== '/';
     };
 
-    _.each(ast.stylesheet.rules, function(rule, ruleIndex) {
-      var basePath = pathDirname(rule.position.source);
-
-      // Set the correct basePath based on how the linked asset will be served.
-      // XXX This is wrong. We are coupling the information about how files will
-      // be served by the web server to the information how they were stored
-      // originally on the filesystem in the project structure. Ideally, there
-      // should be some module that tells us precisely how each asset will be
-      // served but for now we are just assuming that everything that comes from
-      // a folder starting with "/packages/" is served on the same path as
-      // it was on the filesystem and everything else is served on root "/".
-      if (! basePath.match(/^\/?packages\//i))
-          basePath = "/";
-
-      _.each(rule.declarations, function(declaration, declarationIndex) {
-        var parts, resource, absolutePath, quotes, oldCssUrl, newCssUrl;
-        var value = declaration.value;
-
-        // Match css values containing some functional calls to `url(URI)` where
-        // URI is optionally quoted.
-        // Note that a css value can contains other elements, for instance:
-        //   background: top center url("background.png") black;
-        // or even multiple url(), for instance for multiple backgrounds.
-        var cssUrlRegex = /url\s*\(\s*(['"]?)(.+?)\1\s*\)/gi;
-        while (parts = cssUrlRegex.exec(value)) {
-          oldCssUrl = parts[0];
-          quotes = parts[1];
-          resource = url.parse(parts[2]);
-
-          // Rewrite relative paths to absolute paths.
-          // We don't rewrite urls starting with a protocol definition such as
-          // http, https, or data.
-          if (isRelative(resource.path) && resource.protocol === null) {
-            absolutePath = pathJoin(basePath, resource.path);
-            newCssUrl = "url(" + quotes + absolutePath + quotes + ")";
-            value = value.replace(oldCssUrl, newCssUrl);
-          }
+    var rewriteRules = function (rules) {
+      _.each(rules, function(rule, ruleIndex) {
+
+        // do we need to recurse?
+        if (rule.rules != null) {
```

Comment:
What does this check mean? Can we make it more specific? The comment above isn't very descriptive either.